components/hire/alumni.json

```
@@ -471,10 +471,34 @@
       "CSS",
       "Sass",
       "Bootstrap",
-      "Git" ,
+      "Git",
       "Firebase"
     ],
     "status": "looking for jobs",
     "graduationDate": "29-01-2020"
+  },
+  {
+    "id": 31,
+    "roles": ["alumni"],
+    "name": "Axel Leonide Gassay Ekamba",
+    "email": "axelleonide@gmail.com",
+    "photo": "/static/alumni/photos/Axel.jpg",
```

```suggestion
    "photo": "/static/alumni/photos/axel.jpg",
```

You called the image axel.jpg lowercase a

---

thanks for your feed back @benna100 , I have fixed it.